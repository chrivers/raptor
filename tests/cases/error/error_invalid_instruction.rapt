# line 1
FLORG --foo "bar"
$ for x in [1,2,3,4,5]
RUN ls -l
$ endfor
# line 4
