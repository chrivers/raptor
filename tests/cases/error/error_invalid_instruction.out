error: parsing error: expected FILE
 --> tests/cases/error/error_invalid_instruction.rapt:1:1
  |
1 | FLORG --foo "bar"
  | ^^^^^^^^^^^^^^^^^ expected FILE
2 | RUN ls -l
3 | RUN ls -l
4 | RUN ls -l
5 | RUN ls -l
6 | RUN ls -l
7 |
  |
